### ISSUE TITLE
Field selectedCountryCode doesn't work on Android

### ISSUE THREAD

**Issue by dulciejackson** [persona-id/inquiry-android-2#1]
Using react-native-persona v2.2.10 (Android SDK 2.2.21)

I am trying to pass a pre-filled country code through using Persona, so that the country picker defaults to United Kingdom rather than United States. Using the react-native-persona package this works for iOS, but Android still shows United States as the default.

Code example:
```
Inquiry.fromTemplate(PERSONA_TEMPLATE_ID)
        .environment(PERSONA_ENVIRONMENT as Environment)
        .fields(
          Fields.builder()
            .string("selectedCountryCode", "GB")
            .build(),
        )
...
```

**Comment by SalvatoreT** [comment]
Hey @dulciejackson, we're taking a look! Thank you for raising the issue.

**Comment by SalvatoreT** [comment]
@dulciejackson, we just shipped the [React Native SDK v2.2.11](https://docs.withpersona.com/docs/react-native-sdk-v2-changelogs#v2211---2022-05-25) which fixed the issue you describe here.

Thank you for your patience, and please verify this worked for you when you get the chance.
